From 96db599c15a4a00d986a9b744921944f78d09bc7 Mon Sep 17 00:00:00 2001
From: google <android@google.com>
Date: Tue, 28 Mar 2023 16:29:20 -0700
Subject: [PATCH] arm64: compat: Move sigreturn32.S to .rodata section

---
 arch/arm64/kernel/sigreturn32.S | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/kernel/sigreturn32.S b/arch/arm64/kernel/sigreturn32.S
index 6ecda4d..d4a13de 100644
--- a/arch/arm64/kernel/sigreturn32.S
+++ b/arch/arm64/kernel/sigreturn32.S
@@ -28,6 +28,7 @@
 
 #include <asm/unistd.h>
 
+	.section .rodata
 	.globl __aarch32_sigret_code_start
 __aarch32_sigret_code_start:
 
-- 
2.7.4

